docs/source/guide/pt-4-low-level.md

```diff
@@ -0,0 +1,118 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# What happens when I use PT?
+
+The workflow of Pauli Twirling (PT) in Mitiq is represented in the figure below.
+
+```{figure} ../img/pt_workflow.svg
+---
+width: 700px
+name: pt-workflow-overview
```

Comment:
I think this might have to be renamed to something like `pt-workflow-overview2` because `pt-workflow-overview` is used in the main page `pt.md`.

Reply:
@purva-thakre That's by design, since we're using the same image for both places in the docs. Take a look [here](https://mitiq--1848.org.readthedocs.build/en/1848/guide/pt-4-low-level.html) and we can see the code execution errors.

That said, I keep finding inaccuracies (both in the code and docs/theory), so this one isn't quite ready for review yet (when I marked it as so late last night I was apparently rather hopeful 🤣)!

Reply:
Thanks @purva-thakre, should be all set now!